### PR TITLE
Enhancement of QLed.

### PR DESCRIPTION
**Pull request by fernandohds564**
Add selected property to be used as interaction mechanism with mouse or keyboard selections.

### CODE REVIEW

File: pyqt-apps/siriushla/widgets/QLed.py [hunk 1]
```diff
@@ -13,274 +13,59 @@
                         QRectF, Property, Q_ENUMS
 from qtpy.QtSvg import QSvgRenderer
 
-SHAPE = {'Circle': 1, 'Round': 2, 'Square': 3, 'Triangle': 4}
+__path__ = _os.path.dirname(__file__)
+
+
+class ShapeMap:
+    """Shape enum mapping class."""
+    Circle = 1
```
Comment (anacso17) [inline pyqt-apps/siriushla/widgets/QLed.py]: Missing blank line after class docstring

File: pyqt-apps/siriushla/widgets/QLed.py [hunk 2]
```diff
@@ -438,6 +234,14 @@ def toggleValue(self):
         self.m_state = 0 if self.m_state else 1
         self.update()
 
+    def isSelected(self):
+        return self._isselected
```
Comment (anacso17) [inline pyqt-apps/siriushla/widgets/QLed.py]: Missing docstring in public method

File: pyqt-apps/siriushla/widgets/QLed.py [hunk 3]
```diff
@@ -438,6 +234,14 @@ def toggleValue(self):
         self.m_state = 0 if self.m_state else 1
         self.update()
 
+    def isSelected(self):
```
Comment (anacso17) [inline pyqt-apps/siriushla/widgets/QLed.py]: Missing docstring in public method

File: pyqt-apps/siriushla/widgets/QLed.py [hunk 4]
```diff
@@ -438,6 +234,14 @@ def toggleValue(self):
...
+        return self._isselected
+
+    def setSelected(self, sel):
```
Comment (anacso17) [inline pyqt-apps/siriushla/widgets/QLed.py]: Missing docstring in public method

**Comment by anacso17** [comment]
Great improvements @fernandohds564 !

**Comment by fernandohds564** [comment]
Thanks for the suggestions @anacso17 ! I've addressed them in the last commit (5aafe26).
I will merge this PR.

**Pull request merged**